src/olympia/lib/settings_base.py

```diff
@@ -338,46 +333,44 @@ def lazy_langs(languages):
                 'olympia.amo.context_processors.static_url',
                 'jingo_minify.helpers.build_ids',
             ),
-            'loaders': (
-                'olympia.lib.template_loader.Loader',
-                'django.template.loaders.filesystem.Loader',
-                'django.template.loaders.app_directories.Loader',
-            )
+            'extensions': (
+                'jinja2.ext.autoescape',
+                'jinja2.ext.do',
+                # 'jinja2.ext.i18n',
```

Comment:
oh, let me see if this is needed or not. Might just be able to remove that ext, I think it conflicts with something…